src/CoreWCF.NetTcp/tests/Services/TestService.cs

```diff
@@ -93,5 +93,17 @@ public string EchoForImpersonation(string value)
         {
             return value;
         }
+
+        [AuthorizeRole(AllowedRoles = "InvalidRole, CoreWCFGroupAdmin")]
+        public string EchoForAuthorizarionOneRole(string value)
```

Comment:
Is this the only way to provide multiple roles? Or can I apply the attribute multiple times? Ie:
```c#
        [AuthorizeRole(AllowedRoles = "InvalidRole")]
        [AuthorizeRole(AllowedRoles = "CoreWCFGroupAdmin")]
        public string EchoForAuthorizarionOneRole(string value)
        {
            return value;
        }
```

Reply:
Not allowing multiple attributes at this time... just trying to make it similar to asp.net core